src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import HeroImage from "../../assets/hero-img.jpg"
+import "./Hero.scss"
+
+export default function Hero() {
+    return(
+        <div className="hero">
+            <div className="hero-image" style={{ backgroundImage: `url(${HeroImage})` }}></div>
+            <div className="container">
+                <form action="#" className="search-form">
```

Comment:
Hi Natalia,
please consider is it possible to decompose the component into other separate standalone components.

Reply:
Thanks, I changed the structure and created SearchForm component)